src/tests/contract/ethlance_test.cljs

```diff
@@ -6,44 +6,112 @@
             [district.server.web3 :refer [web3]]
             [ethlance.server.contract.ethlance :as ethlance]
             [ethlance.shared.smart-contracts-dev :as addresses]
+            ["fs" :as fs]
 
-            [cljs-web3-next.eth :as web3-eth]
             [district.server.smart-contracts :as smart-contracts]
-            [cljs.core.async :refer [go <!]]
+            [cljs.core.async :refer [<! put! chan go]]
             [district.shared.async-helpers :refer [<?]]))
 
+(defn publish-to-channel-callback [&stuff])
+(defn subscribe->chan [contract method options]
+  (let [out (chan)
+        callback (fn [event]
+                   (println "Got an event" event)
+                   (put! out event)
+                   )
+        ]
+    (smart-contracts/subscribe-events contract metohd options [publish-to-channel-callback])
```

Comment:
This line produces the exception (even with empty callback implementation as it is). Removing it gets rid of the exception (but also of any useful work :( )

The exception stack trace:
```
Testing tests.contract.ethlance-test

/home/madis/code/district0x/ethlance/server/tests/cljs-runtime/cljs/core.cljs:9524
      (throw (js/Error. (str "Doesn't support name: " x))))))
      ^
Error: Doesn't support name: 
    at Object.cljs$core$name [as name] (/home/madis/code/district0x/ethlance/server/tests/cljs-runtime/cljs/core.cljs:9524:7)
    at Function.cljs$core$IFn$_invoke$arity$variadic (/home/madis/code/district0x/ethlance/server/tests/cljs-runtime/cljs_web3_next/eth.cljs:34:1)
    at Object.district$server$smart_contracts$subscribe_events [as subscribe_events] (/home/madis/code/district0x/ethlance/server/tests/cljs-runtime/district/server/smart_contracts.cljs:170:3)
    at Object.tests$contract$ethlance_test$subscribe__GT_chan [as subscribe__GT_chan] (/home/madis/code/district0x/ethlance/server/tests/cljs-runtime/tests/contract/ethlance_test.cljs:17:3)
    at inst_45431 (/home/madis/code/district0x/ethlance/server/tests/cljs-runtime/tests/contract/ethlance_test.cljs:37:12)
    at switch__35204__auto__ (/home/madis/code/district0x/ethlance/server/tests/cljs-runtime/tests/contract/ethlance_test.cljs:37:12)
    at Function.tests$contract$ethlance_test$state_machine__35205__auto____1 [as cljs$core$IFn$_invoke$arity$1] (/home/madis/code/district0x/ethlance/server/tests/cljs-runtime/tests.contract.ethlance_test.js:386:4)
    at Object.fexpr__40437 [as run_state_machine] (/home/madis/code/district0x/ethlance/server/tests/cljs-runtime/cljs/core/async/impl/ioc_helpers.cljs:43:3)
    at Object.cljs$core$async$impl$ioc_helpers$run_state_machine_wrapped [as run_state_machine_wrapped] (/home/madis/code/district0x/ethlance/server/tests/cljs-runtime/cljs/core/async/impl/ioc_helpers.cljs:45:1)
    at /home/madis/code/district0x/ethlance/server/tests/cljs-runtime/cljs/core/async/impl/ioc_helpers.cljs:56:38
    at G__42453 (/home/madis/code/district0x/ethlance/server/tests/cljs-runtime/cljs_promises/async.cljs:30:31)
    at Immediate.cljs$core$async$impl$dispatch$process_messages (/home/madis/code/district0x/ethlance/server/tests/cljs-runtime/cljs/core/async/impl/dispatch.cljs:26:7)
    at processImmediate (internal/timers.js:443:21)
    at process.topLevelDomainCallback (domain.js:136:23)
```